### PR TITLE
stranslate: Add version 1.1.2.509

### LINKED ISSUE #13196: [Request]: STranslate
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

STranslate

### Description

A ready-to-use, ready-to-go translation ocr tool developed by WPF

### Homepage

https://stranslate.zggsong.com

### Download Link(s)

https://github.com/ZGGSONG/STranslate/releases/download/1.1.0.424/STranslate_1.1.0.424_win-x64.zip

### Some Indication of Popularity/Repute

https://github.com/ZGGSONG/STranslate

### PR DESCRIPTION
**Pull request by ZGGSONG**
Closes #13196

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/stranslate.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "license": "MIT",
+    "version": "1.1.0.424",
+    "url": "https://github.com/ZGGSONG/STranslate/releases/download/1.1.0.424/STranslate_1.1.0.424_win-x64.zip",
+    "hash": "b5c165c414ed6a6e2b7439da7a7e6a31c7f994de4514523d315df545b51c9771",
+    "shortcuts": [
+        [
+            "STranslate.exe",
+            "STranslate"
+        ]
+    ],
```
Comment (aliesbelik) [inline bucket/stranslate.json:13]:
```suggestion
    ],
    "post_install": "Remove-Item \"$dir\\Updater*\" -Force -Recurse",
```

**Review by aliesbelik — COMMENTED** [review]
Guess built-in updater could be removed since updates will be managed by scoop.

**Comment by ZGGSONG** [comment]
> Guess built-in updater could be removed since updates will be managed by scoop.

When I didn't understand 'scoop', I searched for a long time to find out what upgrade plan to use. Finally, I referenced the upgrade plan of another software, GitHub API. If I had understood 'scoop' from the beginning, I would definitely have chosen 'scoop' without hesitation. It's really great.

**Pull request merged**